Makefile

```diff
@@ -48,6 +48,11 @@ container-install:
 	sudo docker build . -f Dockerfile.build -t runnc-build
 	sudo docker run --rm -v /opt/runnc/:/opt/runnc/ -v /usr/local/bin:/usr/local/bin -v ${PWD}:/go/src/github.com/nabla-containers/runnc -w /go/src/github.com/nabla-containers/runnc runnc-build make install
 
+container-uninstall:
+	sudo docker rmi runnc-build | true
```

Comment:
I think `sudo docker rmi -f` works here.

Reply:
Yes, it does.